tony-history-server/conf/application.example.conf

```diff
@@ -1,9 +1,10 @@
 play.http.secret.key = "test"
 play.modules.enabled += "Startup"
-keytab {
-  user = "abc@example.com"
-  location = "/path/to/abc.keytab"
-}
+
 tony {
-  historyFolder = "/path/to/history/folder/"
+  keytab {
+    user = "user1"
+    location = "/some/path/to/tony.keytab"
+  }
+  history.location = "/Users/phattran/.tony/system"
```

Comment:
Can we make the example `tony.history.location` more generic, something like `/tmp/tony-history`?

Reply:
Since this is already merged, I'm going to include this in https://github.com/linkedin/TonY/pull/103.